html-aam/index.html

```diff
@@ -2251,38 +2251,45 @@ <h4 id="el-figcaption">`figcaption`</h4>
               </th>
               <td>
-                <div class="role"><span class="type">Roles:</span> `ROLE_SYSTEM_TEXT`; `IA2_ROLE_CAPTION`</div>
-                <div class="relations"><span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element</div>
+                <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
+                <div class="relations">
+                  <span class="type">Relations:</span>
+                  If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `IA2_RELATION_DETAILS_FOR` with parent `figure`.
+                </div>
               </td>
             </tr>
             <tr>
               <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
               <td>
-                <div class="ctrltype"><span class="type">Control Type:</span> `Text`</div>
+                <div class="ctrltype"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
               </td>
             </tr>
             <tr>
               <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
               <td>
-                <div class="role"><span class="type">Role:</span> `ATK_ROLE_CAPTION`</div>
+                <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
                 <div class="relations">
                   <span class="type">Relations:</span>
-                  `ATK_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+                  If this is the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a> element, then expose `RELATION_DETAILS_FOR` referencing the parent `figure`
+                  element.
                 </div>
               </td>
             </tr>
             <tr>
               <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
               <td>
-                <div class="role"><span class="type">AXRole:</span> `AXGroup`</div>
-                <div class="subrole"><span class="type">AXSubrole:</span> `(nil)`</div>
-                <div class="roledesc"><span class="type">AXRoleDescription:</span> `"group"`</div>
+                <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
               </td>
             </tr>
             <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td>
+                <p>
+                  Only the first child instance of a <a href="#el-figcaption">`figcaption`</a> element that is present in the accessibility tree (e.g., not 'hidden') provides details for its parent
```

Comment:
@jcsteh's [migrated comment](https://github.com/w3c/html-aam/pull/359/files#r1429366035) provided below:

> For figure, we're explicit that aria-labelledby, aria-describedby or aria-details prevent exposure of the details relation. However, we're not similarly explicit here for the reverse relation. Common sense will hopefully prevail, but I don't love relying on common sense for a spec. Is there something I'm missing that would otherwise make this explicit?
> 
> This is a similar situation to https://github.com/w3c/html-aam/pull/481/files#r1424943049. It seems we tend to use ARIA to express roles in a cross-platform manner wherever possible now, but we don't do that for relations, I guess largely because reverse relations and the details-roles attribute complicate things.